qiling/os/windows/variables.py

```diff
@@ -0,0 +1,22 @@
+from qiling.os.windows.const import *
+
```

Comment:
this file should provide default value, but if user defined value should be in rootfs or "c:\" in the sandbox. if user we might want to all a ql.windows_var = filename.py  to allow user to add in the config without changing the core

Reply:
Yeah, this was on point that I needed to discuss. We need something for variables, like User, environment, temp path and so on.
There are many ways to implement it, or a config file that we parse, or something else.
My implementation was "the easy way" for the programmer

Reply:
like i said, we need to have a option just like ql.gdb, we need to have a ql.win_env = file so user can copy the file point to the file. 

easiest method is, user copy this file, move to location and redefine it as
```
ql.win_env = config.py
```

if nothing is being defined. this file will be default.